src/components/DocumentationTopic/Summary/OnThisPageNav.vue

```diff
@@ -1,90 +0,0 @@
-<!--
-  This source file is part of the Swift.org open source project
-
-  Copyright (c) 2021 Apple Inc. and the Swift project authors
-  Licensed under Apache License v2.0 with Runtime Library Exception
-
-  See https://swift.org/LICENSE.txt for license information
-  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
--->
-
-<template>
-  <Section class="on-this-page">
-    <nav aria-labelledby="on-this-page-title">
-      <Title id="on-this-page-title">On This Page</Title>
-      <List>
-        <ListItem v-for="section in sectionsWithFragments" :key="section.anchor">
-          <router-link
-            class="link"
-            :to="buildUrl(section.fragment, $route.query)"
-          >
-            <span class="link-text">{{ section.title }}</span>
-            <span
-              class="icon-holder"
-              aria-hidden="true"
-            >&nbsp;<InlineChevronDownCircleIcon class="link-icon icon-inline" /></span>
-          </router-link>
-        </ListItem>
-      </List>
-    </nav>
-  </Section>
-</template>
-
-<script>
-import InlineChevronDownCircleIcon from 'theme/components/Icons/InlineChevronDownCircleIcon.vue';
-import { buildUrl } from 'docc-render/utils/url-helper';
-import List from './List.vue';
```

Comment:
It's possible that these section/title/list/list-item components can also be deleted now since this might have been the last component using them from the old sidebar (please double check me before you delete them).

Reply:
Great call. Checked, they are no longer used.